brontology/graph/entity/graph.py

```diff
@@ -0,0 +1,22 @@
+# noinspection PyUnresolvedReferences
+from typing import Union, Optional
+
+from brontology.graph.entity.node import Entity, Relation, Lemma
+from brontology.graph.iterable.graph import IterableGraph
+from brontology.relation_extraction.model import TokenRelation
+
+
+class EntityGraph(IterableGraph[Entity, Relation, Lemma]):
+    """A graph that handles `Entity` objects that are connected via `Relation` objects."""
+
+    def __init__(self):
+        super().__init__(Entity, Relation)
+
+    def add_token_relation(self, relation: TokenRelation):
```

Comment:
Das hier ist eine Helfer Methode. Pauschal würde der Graph durch seine Basis `IterableGraph` nur `Lemma`-objekte nehmen, wenn wir etwas hinzufügen wollen.

Mit dieser Methode kann man dann auch `TokenRelation`-Objekte hinzufügen und er wandelt sie dann von selbst in die `Lemma-Objekte um.